README.md

```
@@ -24,7 +24,7 @@ SiliconCompiler is a modular hardware build system ("make for silicon"). The pro
 |**FPGA APR**| VPR, nextpnr, Vivado
 |**Layout Viewer**| Klayout, OpenROAD, Cadence, Synopsys
 |**DRC/LVS**| Klayout, Magic, Synopsys, Siemens
-|**Open PDKs**| sky130, ihp130, gf180, asap7, freepdk45, gt2n, interposer — all shipped by [lambdapdk](https://github.com/siliconcompiler/lambdapdk)
+|**Open PDKs**| sky130, ihp130, gf180, asap7, freepdk45, gt2n, icsprout55, interposer — all shipped by [lambdapdk](https://github.com/siliconcompiler/lambdapdk)
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)